packages/base-driver/package.json

```diff
@@ -67,9 +67,6 @@
     "source-map-support": "0.5.21",
     "validate.js": "0.13.1"
   },
-  "devDependencies": {
```

Comment:
ws is used to run e2e tests. What is the reason to remove it?

Reply:
all dev deps live in the monorepo root `package.json`